src/leafdoc.js

```diff
@@ -114,6 +114,13 @@ Leafdoc.prototype.registerDocumentable = function(name, label) {
 	return this;
 };
 
+// 🍂method getTemplateEngine(): Handlebars
+// Returns handlebars template engine used to render templates.
+// You can use it for override helpers or register new.
+Leafdoc.prototype.getTemplateEngine = function() {
```

Comment:
My use case is override default `type` helper to better one in order to support complex types, for example `Array<L.LatLng>`. Like this:
```js
function makeTypesBetter(handlebars) {
  const makeLink = handlebars.helpers.type;
  const typeRe = /[\w\._\s]+/g;

  handlebars.registerHelper('type', function(str) {
    return str
      .replace(/[\<\>]/g, function(str) {
        return escapeHtml(str);
      })
      .replace(typeRe, function(typeStr) {
        return makeLink(typeStr);
      });
  });
}

makeTypesBetter(doc.getTemplateEngine());
```

Btw, we can add support for complex types in a primary codebase, but nonetheless having ability to extend built-in helpers and add new is good idea I think.